foundation-extensions/consent-for-edge-network/README.md

```diff
@@ -1,6 +1,6 @@
 # Consent for Edge Network
 
-The Adobe Experience Platform Consent mobile extension enables consent preferences collection from your mobile app when using the [Adobe Experience Platform Mobile SDK](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/mobile-core) and the [Edge Network extension](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge).
+The Adobe Experience Platform Consent mobile extension enables consent preferences collection from your mobile app when using the [Adobe Experience Platform Mobile SDK](../../using-mobile-extensions/mobile-core) and the [Edge Network extension](../experience-platform-extension).
```

Comment:
`../../using-mobile-extensions/mobile-core` should be -> `../../foundation-extensions/mobile-core`